web/pom.xml

```diff
@@ -10,7 +10,7 @@
         <relativePath>../parent/pom.xml</relativePath>
     </parent>
 
-    <artifactId>${yumArtifactId}</artifactId>
+    <artifactId>emodb-web</artifactId>
```

Comment:
this is here because later versions of maven don't support variable substitution in the artifactId.